### PR TITLE
Add Python Samples

### PR DESCRIPTION
**Pull request by ConnorKirk**
*Description of changes:*
This PR adds a selection of Python code samples for the use case of detecting personal protective equipment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,11 +10,34 @@ You can also aggregate the PPE detection results and analyze them by time and pl
 
 With Amazon Rekognition PPE detection, you receive a detailed analysis of an image, which includes bounding boxes and confidence scores for persons (up to 15 per image) and PPE detected, confidence scores for the body parts detected, and Boolean values and confidence scores for whether the PPE covers the corresponding body part. The following image shows an example of PPE bounding boxes for head cover, hand covers, and face cover annotated using the analysis provided by the Amazon Rekognition PPE detection feature.
 
+## [Demo]()
```
Comment (matteofigus) [inline README.md:13]:
```suggestion
## [Demo](/demo)
```

File: README.md [hunk 2]
```diff
@@ -10,11 +10,34 @@ You can also aggregate the PPE detection results and analyze them by time and pl
...
+### [Stream Processing](stream-processing/ppe-stream.py)
+
+This sample demonstrates how to detect protective equipment on a streamed video
+
 ## Amazon Rekognition DetectProtectiveEquipment API
 
 To detect PPE in an image, you call the DetectProtectiveEquipment API and pass an input image. You can provide the input image (in JPG or PNG format) either as raw bytes or as an object stored in an Amazon Simple Storage Service (Amazon S3) bucket. You can optionally use the SummarizationAttributes (ProtectiveEquipmentSummarizationAttributes) input parameter to request summary information about persons that are wearing the required PPE, not wearing the required PPE, or are indeterminate.
 
-
 # Navigate
```
Comment (matteofigus) [inline README.md:41]: Does the previous list make this "Navigate" section obsolete?

**Review by matteofigus — CHANGES_REQUESTED** [review]
Just a minor comment

File: README.md (hunk 2 above)
Comment (ConnorKirk) [reply README.md:41]: I somehow missed the existing navigation section 🤦 
Should I move the entries to the navigation section as well, or remove the navigation section? 

My preference would be to have just one, but not both. I don't mind which

Comment (matteofigus) [reply README.md:41]: Agree, yes I have no preference either

Comment (ConnorKirk) [reply README.md:41]: I will remove the navigation section then.

**Review by matteofigus — APPROVED** [review]

**Pull request merged**